pokete_classes/npcs.py

```diff
@@ -256,11 +255,49 @@ def action(self):
                 self.text({True: self.lose_texts,
                            False: self.win_texts + ["Here's $20!"]}
                           [is_winner])
-                heal(self) if is_winner else None
+                self.heal() if is_winner else None
```

Comment:
I think you should just do `if is_winner: self.heal()`, using the ternary if for non-expressions is not ideal